### ISSUE TITLE
Session is forgotten in between steps

### ISSUE THREAD

**Issue by cleb** [Behatch/contexts#215]
Using behatch together with symfony 1.4 and knplabs/friendly-contexts. Whenever we execute a step that uses behatch, it logs out of the system (the session seems to be lost)

I *think* it is caused by the header reset in 4cc6cfcab8914a76403dc044b450cb3a91e0651a

**Comment by sanpii** [comment]
Yes, now the headers are reset between scenarios. You need to login before each scenario, see [background](http://behat.org/en/latest/user_guide/writing_scenarios.html#backgrounds) section.

**Comment by cleb** [comment]
Resetting between scenarios is the expected behavior. However, it resets between each and every step

**Comment by sanpii** [comment]
> should I try to come up with a fix for this?

I can try to find the bug if you have a simple example to reproduce its.

**Comment by cleb** [comment]
I will try to come up with an example (I will have to write it from scratch so as not to compromise the IP of the company I work for so it might take me a while to do it)

**Comment by brice** [comment]
I think i encounter the same problem. Did you find a solution?
